lib/ansible/modules/windows/win_updates.py

```diff
@@ -58,6 +58,20 @@
         - This is only used if C(reboot=yes) and a reboot is required.
         default: 1200
         version_added: '2.5'
+    server_selection:
+        description:
+        - Defines the Windows Update source catalog.
+        - C(default) Use the default search source.  For many systems default is
+          set to the Microsoft Windows Update catalog.
```

Comment:
I would also point out would default to a managed server catalog if defined otherwise would be a Windows Update catalog.